python/avi/migrationtools/setup.py

```
@@ -33,7 +34,7 @@
                       'pycrypto', 'ecdsa', 'pyOpenssl', 'nose-html-reporting',
                       'nose-testconfig', 'ConfigParser', 'xlsxwriter', 'jinja2',
                       'pandas', 'openpyxl', 'appdirs',  'pexpect',
-                      'unittest2', 'networkx'],
+                      'unittest2', 'networkx', 'f5-sdk', 'bigsuds'],
```

we cannot have these tools be installed by avi. We need to raise proper error if f5-sdk and bigsuds is not found.

---

so we need to through exceptions at run time if the packages or not found , isn't it ?

---

this is back... we should remove 'f5-sdk', 'bigsuds' dependencies